src/components/ItemDetail.js

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react';
+import { ListContext } from '../context/ListContext';
+import { useParams, Link } from 'react-router-dom';
+import { ArrowBackIcon } from '@chakra-ui/icons';
+
+export default function ItemDetail() {
+  let currentList = useContext(ListContext);
+
+  let { id } = useParams();
+
+  let currentItem = currentList.userList.find((item) => item.id == id);
+
+  // determine number of days since last purchase date
+  const timeNow = new Date().getTime() / 1000;
+  console.log(timeNow);
+  let timeSinceLastPurchase;
+  // if lastPurchased dosen't exist, set daysUntilPurchase = today-date created
+  if (currentItem.lastPurchased) {
+    timeSinceLastPurchase = Math.ceil(
+      (timeNow - currentItem.lastPurchased.seconds) / (24 * 60 * 60),
+    );
+  } else {
+    timeSinceLastPurchase = 'N/A';
+  }
+  console.log(currentItem);
+
+  return (
+    <div>
+      <Link to={'/view-list'} aria-label="back arrow, return to list view">
+        <ArrowBackIcon />
+      </Link>
+      <ul>
+        <li> Name: {currentItem.itemName} </li>
+        <li> Days Since Last Purchased: {timeSinceLastPurchase} </li>
+        <li>
+          {' '}
+          Estimated Days Until Next Purchase: {
+            currentItem.daysUntilPurchase
```

Comment:
With the way the logic is written, will this be a negative number? Do we want it to be, so that we can see how "overdue" the item is? or should we cap at 0?

Reply:
This could be addressed in the styling stories? Maybe have the text in red if it is a negative number?